src/prototypecomponentsnavigatorwindow/prototypecomponentsnavigatorwindow.h

```diff
@@ -0,0 +1,38 @@
+#ifndef PROTOTYPECOMPONENTSNAVIGATORWINDOW_H
+#define PROTOTYPECOMPONENTSNAVIGATORWINDOW_H
+
+#include <QWidget>
+
+class QAbstractItemModel;
+
+namespace Ui {
+class PrototypeComponentsNavigatorWindow;
+}
+
+class PrototypeComponentsNavigatorWindow : public QWidget
+{
+    Q_OBJECT
+
+public:
+    explicit PrototypeComponentsNavigatorWindow(QWidget *parent = nullptr);
+    ~PrototypeComponentsNavigatorWindow();
+
+public:
+    void setModel(QAbstractItemModel*);
+
+private:
+    Ui::PrototypeComponentsNavigatorWindow *ui;
+
+signals:
```

Comment:
Move "signals" and "private slots"  before private section

Reply:
done